sopel/bot.py

```diff
@@ -417,6 +426,13 @@ def reply(self, message, destination=None, reply_to=None, notice=False):
                 reply_to = self._trigger.nick
             self._bot.reply(message, destination, reply_to, notice)
 
+        def kick(self, nick, channel=None, message=None):
+            if channel is None:
+                channel = self._trigger.sender
```

Comment:
Easiest way to match behavior to the desired outcome described in the docstring:

```suggestion
                if self._trigger.is_privmsg:
                    return
                else:
                    channel = self._trigger.sender
```